src/olympia/promoted/tasks.py

```diff
@@ -6,30 +6,33 @@
 from olympia.amo.celery import task
 from olympia.amo.decorators import use_primary_db
 from olympia.constants.promoted import NOT_PROMOTED, NOTABLE
+from olympia.reviewers.models import UsageTier
 from olympia.versions.utils import get_staggered_review_due_date_generator
-from olympia.zadmin.models import get_config
 
 from .models import PromotedAddon
 
 
 log = olympia.core.logger.getLogger('z.promoted.tasks')
 
-NOTABLE_ADU_LIMIT_CONFIG_KEY = 'notable-adu-threshold'
+NOTABLE_TIER_SLUG = 'notable'
 
 
 @task
 @use_primary_db
 def add_high_adu_extensions_to_notable():
     """Add add-ons with high ADU to Notable promoted group."""
-    adu_limit = get_config(NOTABLE_ADU_LIMIT_CONFIG_KEY)
-    if not adu_limit:
+    try:
+        lower_adu_threshold = UsageTier.objects.get(slug='notable').lower_adu_threshold
```

Comment:
Should have picked this up at the time, but we're not using the constant `NOTABLE_TIER_SLUG` here 🙃